jwst/pipeline/calwebb_detector1.py

```diff
@@ -87,6 +89,7 @@ def process(self, input):
 
             input = self.group_scale(input)
             input = self.dq_init(input)
+            input = self.emicorr(input)
```

Comment:
So obviously this is going to be called when processing any/all data sets. That means all non-MIRI datasets will result in a skip and warning. Any plans to have other instruments update their `calwebb_detector1` parameter ref files in CRDS to just skip this step to begin with, rather than having the step try to execute all the time?

Reply:
good point, I will raise this in the CRDS Jira ticket.

Reply:
Oh wait, because github was only showing a few lines of code, I didn't notice that this was in the section of steps that're only called for MIRI data, so there's no need to worry about it being called unnecessarily for other instruments. I hereby retract my previous comment.